src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day_mon calendar--month-length_28">
```

Comment:
You should use here `<ul>` tag with `<li>` to use more semantic HTML

```suggestion
    <ul class="calendar calendar--start-day_mon calendar--month-length_28">
      <li class="calendar__day-1"></li>
      <li class="calendar__day-2"></li>
      <li class="calendar__day-3"></li>
```